From ed90e67fa168dc0be294f295e4e33062926b8ba0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Burkhard=20L=C3=BCck?= <lueck@hube-lueck.de>
Date: Wed, 19 Apr 2017 07:41:39 +0200
Subject: [PATCH 31/97] Add X-DocPath to settings desktop files

add X-DocPath=choqok/index.html to open the handbook using Help button in settingsdialog

Differential Revision: https://phabricator.kde.org/D5484
---
 choqok/config/accounts/choqok_accountsconfig.desktop     | 1 +
 choqok/config/appearance/choqok_appearanceconfig.desktop | 1 +
 choqok/config/behavior/choqok_behaviorconfig.desktop     | 1 +
 choqok/config/plugins/choqok_pluginconfig.desktop        | 1 +
 4 files changed, 4 insertions(+)

diff --git a/choqok/config/accounts/choqok_accountsconfig.desktop b/choqok/config/accounts/choqok_accountsconfig.desktop
index 07ab62c3..a8c0632c 100644
--- a/choqok/config/accounts/choqok_accountsconfig.desktop
+++ b/choqok/config/accounts/choqok_accountsconfig.desktop
@@ -8,6 +8,7 @@ X-KDE-FactoryName=kcm_choqok_accountsconfig
 X-KDE-ParentApp=choqok
 X-KDE-ParentComponents=choqok
 X-KDE-Weight=20
+X-DocPath=choqok/index.html
 
 Name=Accounts
 Name[ar]=الحسابات
diff --git a/choqok/config/appearance/choqok_appearanceconfig.desktop b/choqok/config/appearance/choqok_appearanceconfig.desktop
index 1f57242a..14127727 100644
--- a/choqok/config/appearance/choqok_appearanceconfig.desktop
+++ b/choqok/config/appearance/choqok_appearanceconfig.desktop
@@ -7,6 +7,7 @@ X-KDE-Library=kcm_choqok_appearanceconfig
 X-KDE-FactoryName=kcm_choqok_appearanceconfig
 X-KDE-ParentApp=choqok
 X-KDE-ParentComponents=choqok
+X-DocPath=choqok/index.html
 
 Name=Appearance
 Name[ar]=المظهر
diff --git a/choqok/config/behavior/choqok_behaviorconfig.desktop b/choqok/config/behavior/choqok_behaviorconfig.desktop
index 567281d4..1f23f790 100644
--- a/choqok/config/behavior/choqok_behaviorconfig.desktop
+++ b/choqok/config/behavior/choqok_behaviorconfig.desktop
@@ -8,6 +8,7 @@ X-KDE-FactoryName=kcm_choqok_behaviorconfig
 X-KDE-ParentApp=choqok
 X-KDE-ParentComponents=choqok
 X-KDE-Weight=10
+X-DocPath=choqok/index.html
 
 Name=Behavior
 Name[ar]=السّلوك
diff --git a/choqok/config/plugins/choqok_pluginconfig.desktop b/choqok/config/plugins/choqok_pluginconfig.desktop
index e6f1365b..1c9e558c 100644
--- a/choqok/config/plugins/choqok_pluginconfig.desktop
+++ b/choqok/config/plugins/choqok_pluginconfig.desktop
@@ -8,6 +8,7 @@ X-KDE-FactoryName=kcm_choqok_pluginconfig
 X-KDE-ParentApp=choqok
 X-KDE-ParentComponents=choqok
 X-KDE-Weight=200
+X-DocPath=choqok/index.html
 
 Name=Plugins
 Name[ar]=الملحقات
-- 
2.17.1

